<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficadvisorysystemClass</class>
 <widget class="QWidget" name="TrafficadvisorysystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Trafficadvisorysystem</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>680</width>
     <height>460</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>631</width>
       <height>280</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="queryButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>62</y>
       <width>160</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Query</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="PriceradioButton">
     <property name="geometry">
      <rect>
       <x>161</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="TimeradioButton">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>47</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Time Costing</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Strategy</string>
     </property>
    </widget>
    <widget class="QLabel" name="trans">
     <property name="geometry">
      <rect>
       <x>231</x>
       <y>100</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="start">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>117</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="select_01">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>113</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>select</string>
     </property>
    </widget>
    <widget class="QPushButton" name="select_02">
     <property name="geometry">
      <rect>
       <x>391</x>
       <y>110</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>select</string>
     </property>
    </widget>
    <widget class="QLabel" name="end">
     <property name="geometry">
      <rect>
       <x>321</x>
       <y>115</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>end</string>
     </property>
    </widget>
    <widget class="QComboBox" name="queryMode">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Train</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Plane</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>631</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delButton">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>390</y>
       <width>90</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>563</x>
       <y>390</y>
       <width>90</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>390</y>
       <width>130</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="changeButton">
     <property name="geometry">
      <rect>
       <x>362</x>
       <y>390</y>
       <width>90</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>90</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>461</x>
       <y>390</y>
       <width>90</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QComboBox" name="typechoose">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>130</width>
       <height>35</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>train</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>plane</string>
      </property>
     </item>
    </widget>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>130</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Traffic Advisory System</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Query</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="trafficadvisorysystem.qrc"/>
 </resources>
 <connections/>
</ui>
